src/main/java/com/ldtteam/structurize/placement/handlers/placement/PlacementHandlers.java

```diff
@@ -708,11 +708,21 @@ public boolean canHandle(@NotNull final World world, @NotNull final BlockPos pos
         public ActionProcessingResult handle(
           @NotNull final World world,
           @NotNull final BlockPos pos,
-          @NotNull final BlockState blockState,
+          @NotNull final BlockState blockStateIn,
           @Nullable final CompoundNBT tileEntityData,
           final boolean complete,
           final BlockPos centerPos)
         {
+            // mojang abusing lazyupdates, this modification always happens, but we need it now, not later
+            BlockState blockState = blockStateIn;
+            if (blockState.getBlock() instanceof HopperBlock)
+            {
+                boolean flag = !world.isBlockPowered(pos);
+                if (flag != blockState.get(HopperBlock.ENABLED))
```

Comment:
this should be its own handler then.

Reply:
Meh, not gonna copy entire container handler because of one block being dumb, generally we need redstone sync code which would probably fix this too

Reply:
well, you can extend and override it.

Reply:
Pff, k